Bottom: 2d950be548f8e407a8ba05e93fe9e96c61aa85ed
Top:    64a32b8e271590bd526ac441a9a232dbbf3ca21f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 21:26:54 +0800

Refresh of vhost_net-batch-update-used

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index bc66768..367d802 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -451,7 +451,7 @@ static int vhost_net_tx_get_vq_desc(struct vhost_net *net,
 				  out_num, in_num, NULL, NULL);
 
 	if (r == vq->num && vq->busyloop_timeout) {
-		if (vhost_sock_zcopy(vq->private_data))
+		if (!vhost_sock_zcopy(vq->private_data))
 			vhost_net_signal_used(nvq);
 		preempt_disable();
 		endtime = busy_clock() + vq->busyloop_timeout;
